chainerx_cc/chainerx/python/routines.cc

```diff
@@ -275,6 +275,20 @@ void InitChainerxCreation(pybind11::module& m) {
           py::arg("endpoint") = true,
           py::arg("dtype") = nullptr,
           py::arg("device") = nullptr);
+    m.def("meshgrid", [](py::args xi, py::kwargs kwargs) {
+        std::vector<Array> xs;
+        std::string indexing("xy");
```

Comment:
Could you use list initialization? Just a better practice. https://stackoverflow.com/questions/18222926/why-is-list-initialization-using-curly-braces-better-than-the-alternatives
```suggestion
        std::string indexing{"xy"};
```

Reply:
oops sorry. I tend to miss that. Fixed.